from mysql.connector import connect, Error

class userDB:

    def __init__(self):

        with open("config_sql", "r") as file:
            configs = file.readlines()
            file.close()
        with open("sqlpassword", "r") as file:
            sqlpassword = file.read().strip()
            file.close()

        self.connection = connect(host=str(configs[0]).strip(), user=str(configs[1]).strip(), password=sqlpassword, database="userinfotest")
        self.cursor = self.connection.cursor()

    def close_down(self):
        self.cursor.close()
        self.connection.close()
        pass


class ProjectDB(userDB):
    def __init__(self):

        with open("config_sql", "r") as file:
            configs = file.readlines()
            file.close()
        with open("sqlpassword", "r") as file:
            sqlpassword = file.read().strip()
            file.close()

        self.connection = connect(host=str(configs[0]).strip(), user=str(configs[1]).strip(), password=sqlpassword, database="projects")
        self.cursor = self.connection.cursor()
